js/zonda.js

```diff
@@ -962,258 +1376,79 @@ module.exports = class zonda extends Exchange {
-        //         "items":[
-        //             ["1591503060000",{"o":"0.02509572","c":"0.02509438","h":"0.02509664","l":"0.02509438","v":"0.02082165","co":"17"}],
-        //             ["1591503120000",{"o":"0.02509606","c":"0.02509515","h":"0.02509606","l":"0.02509487","v":"0.04971703","co":"13"}],
-        //             ["1591503180000",{"o":"0.02509532","c":"0.02509589","h":"0.02509589","l":"0.02509454","v":"0.01332236","co":"7"}],
-        //         ]
-        //     }
-        //
-        const items = this.safeValue (response, 'items', []);
-        return this.parseOHLCVs (items, market, timeframe, since, limit);
-    }
-
-    parseTrade (trade, market = undefined) {
-        //
-        // createOrder trades
-        //
-        //     {
-        //         "rate": "0.02195928",
-        //         "amount": "0.00167952"
-        //     }
-        //
-        // fetchMyTrades (private)
-        //
-        //     {
-        //         amount: "0.29285199",
-        //         commissionValue: "0.00125927",
-        //         id: "11c8203a-a267-11e9-b698-0242ac110007",
-        //         initializedBy: "Buy",
-        //         market: "ETH-EUR",
-        //         offerId: "11c82038-a267-11e9-b698-0242ac110007",
-        //         rate: "277",
-        //         time: "1562689917517",
-        //         userAction: "Buy",
-        //         wasTaker: true,
-        //     }
-        //
-        // fetchTrades (public)
-        //
-        //     {
-        //          id: 'df00b0da-e5e0-11e9-8c19-0242ac11000a',
-        //          t: '1570108958831',
-        //          a: '0.04776653',
-        //          r: '0.02145854',
-        //          ty: 'Sell'
-        //     }
-        //
-        const timestamp = this.safeInteger2 (trade, 'time', 't');
-        const side = this.safeStringLower2 (trade, 'userAction', 'ty');
-        const wasTaker = this.safeValue (trade, 'wasTaker');
-        let takerOrMaker = undefined;
-        if (wasTaker !== undefined) {
-            takerOrMaker = wasTaker ? 'taker' : 'maker';
-        }
-        const priceString = this.safeString2 (trade, 'rate', 'r');
-        const amountString = this.safeString2 (trade, 'amount', 'a');
-        const feeCostString = this.safeString (trade, 'commissionValue');
-        const marketId = this.safeString (trade, 'market');
-        market = this.safeMarket (marketId, market, '-');
-        const symbol = market['symbol'];
-        let fee = undefined;
-        if (feeCostString !== undefined) {
-            const feeCurrency = (side === 'buy') ? market['base'] : market['quote'];
-            fee = {
-                'currency': feeCurrency,
-                'cost': feeCostString,
-            };
-        }
-        const order = this.safeString (trade, 'offerId');
-        // todo: check this logic
-        let type = undefined;
-        if (order !== undefined) {
-            type = order ? 'limit' : 'market';
+        //      "balance": {
```

Comment:
We have here a huge comment (around 300 lines), and almost all samples there are very similar except for the type. I know it was here before but can you dry it a bit, please? Makes it very hard to move around the code